fromNow/from_now_test.go

```diff
@@ -0,0 +1,57 @@
+package fromNow
+
+import (
+	"testing"
+
+	assert "github.com/stretchr/testify/require"
+)
+
+func TestParseTimeComplete(t *testing.T) {
+	assert := assert.New(t)
+	offset, err := parseTime("1 year 2 months 3 weeks 4 days 5 hours 6 minutes 7 seconds")
+
+	assert.NoError(err, "err should be nil")
+	assert.Equal(offset.years, 1, "they should be equal")
```

Comment:
I know it's not originally your code, but I think these error messages aren't very descriptive. Can we have something that tells me more than simply "they should be equal"?